sympy/matrices/expressions/tests/test_trace.py

```diff
@@ -83,9 +83,8 @@ def test_Trace_doit_deep_False():
     assert Trace(q).doit(deep=False).arg == q
 
 
-@XFAIL
 def test_trace_constant_factor():
-    # Issue 9052: LHS gives Trace(MatMul(A))
+    # Issue 9052
```

Comment:
*shrug* I don't see a huge reason to document precisely how it used to be broken, especially given the *test name*.   But its not worth debating so changed.